### ISSUE TITLE
Docs: Add entries for `microtask` and `macrotask` in the glossary

### ISSUE THREAD

**Issue by JeanMeche** [angular/angular#50293]
### Describe the problem that you experienced

I was looking into #50285 when I wanted to add some links for micro/macrotasks but we don't have any entries in the glossary yet 

Some inpiration: https://developer.mozilla.org/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide#microtasks

**Comment by AndrewKushnir** [comment]
@JeanMeche could we just add a link to the mentioned guide and/or other resource that describes what micro and macro tasks are (vs creating new entries in the glossary)?

**Comment by JeanMeche** [comment]
You mean that, in a way, the subject is a bit complex for the glossary and deserves a longer explanation ?

**Comment by AndrewKushnir** [comment]
> You mean that, in a way, the subject is a bit complex for the glossary and deserves a longer explanation ?

Yeah, correct. Also, this is not an Angular-specific concept, so we can just point to resources that define it (for ex. specs).
